src/App.jsx

```diff
@@ -1,11 +1,33 @@
 import * as React from 'react'
-import Button from './Button'
+import Card from './Card'
+import './App.css'
+import { cards } from './assets/cards-data'
 
 export default () => {
   return (
-    <div>
-      <Button isActive={true}>Active</Button>
-      <Button isActive={false}>Not active</Button>
+    <div className="App">
+      {cards.map(card => {
+        const {
+          title,
+          question,
+          answer,
+          isAnswerVisible,
+          tags,
+          isBookmarked,
+          id,
+        } = card
+        return (
+          <Card
+            title={title}
+            question={question}
+            answer={answer}
+            isAnswerVisible={isAnswerVisible}
+            tags={tags}
+            isBookmarked={isBookmarked}
+            key={id}
+          />
+        )
+      })}
```

Comment:
You can destructure the props right away. That way the code is a bit cleaner.
```suggestion
      {cards.map({
          title,
          question,
          answer,
          isAnswerVisible,
          tags,
          isBookmarked,
          id,
        }  => (
          <Card
            title={title}
            question={question}
            answer={answer}
            isAnswerVisible={isAnswerVisible}
            tags={tags}
            isBookmarked={isBookmarked}
            key={id}
          />
        )
      )}
```